azure-pipelines.yml

```diff
@@ -229,11 +229,13 @@ jobs:
                           /p:SignAssetsDir=$(Build.ArtifactStagingDirectory)\Microsoft.Spark.Binaries\Microsoft.Spark.Worker\
                           /p:SignType=$(_SignType)
         msbuildVersion: 15.0
-
-    - task: NuGetCommand@2
+ 
+    - task: DotNetCoreCLI@2
+      displayName: 'Create NuGet packages'
       inputs:
         command: pack
-        packagesToPack: '$(Build.SourcesDirectory)\src\csharp\Microsoft.Spark.nuspec'
+        projects: '$(solution)'
```

Comment:
How about using `$(Build.SourcesDirectory)\src\csharp\Microsoft.Spark.csproj` and remove `IsPackable` in other csproj files? @eerhardt @safern what's the recommended way here?

Reply:
I prefer this approach as it doesn't require changes every time you add new projects as you usually want them as packages (test project have `IsPackable` set to false by default and for the examples I should move the flag to the build.props).
That's why I also used wildcards in the csproj to include the `jar`s
However, currently the sign steps is not currently using wildcards for the project name so that would need to be changed as well.

Let me know what do you think. Happy to switch to just pack that project directly if that is your preference or you don't see new packaged projects added in the near future.

Reply:
@imback82 - do you envision any more nupkgs in the future? I agree with @alefranz here - it is best to have the .yml file say "run pack", and have the projects themselves decide whether they are packable or not. It allows for more packages without having to update this step, which may be missed.

But if we only ever envision a single NuGet package, then hard-coding the one project to pack would be OK.

Reply:
I think there is another nuget package comming for the worker. I think packages should decide if they are packable or not. Also, that way locally if you want to build packages you can just do, dotnet pack Solution.sln and it will generate the packages that the repo builds.

Reply:
Sounds good. Let's keep it this way. Thanks for the feedbacks.